examples/stubbing-spying__intercept/cypress.config.js

```diff
@@ -4,7 +4,7 @@ module.exports = defineConfig({
   defaultCommandTimeout: 8000,
   retries: {
     runMode: 5,
-    openMode: 1,
+    openMode: 0,
```

Comment:
It looks like the default is already "0" https://docs.cypress.io/guides/guides/test-retries#Global-Configuration